pyproject.toml

```diff
@@ -0,0 +1,18 @@
+[tool.poetry]
+name = "genome-browser"
+version = "0.1.1"
+description = "A lightweight Python 3 graphing API for genomic intervals and genomic features."
+authors = ["Clint Valentine <valentine.clint@gmail.com>"]
+license = "MIT"
+
+[tool.poetry.dependencies]
+python = ">=3.7,<3.11"
+matplotlib = "^3.5.2"
+more-itertools = "^8.13.0"
+scipy = "1.7.3"
```

Comment:
![Screenshot from 2022-05-16 22-52-19](https://user-images.githubusercontent.com/13955209/168710324-a97685f8-89b4-4f99-ba8a-1c9d6825d0a4.png)

scipy 1.7.3 is the latest that doesn't retire Python 3.7.